jupyter_collaboration/handlers.py

```diff
@@ -129,10 +87,90 @@ class YDocWebSocketHandler(WebSocketHandler, JupyterHandler):
        receiving a message.
     """
 
-    saving_document: Optional["asyncio.Task[Any]"]
+    files: Dict[str, FileLoader] = {}
     websocket_server: Optional[JupyterWebsocketServer] = None
     _message_queue: "asyncio.Queue[Any]"
 
+    def __init__(
+        self, app: ServerWebApplication, request: HTTPServerRequest, **kwargs: Dict[str, Any]
+    ):
+        super().__init__(app, request, **kwargs)
+
+        # CONFIG
+        file_id_manager = self.settings["file_id_manager"]
+        ystore_class = self.settings["collaborative_ystore_class"]
+        self._cleanup_delay = self.settings["collaborative_document_cleanup_delay"]
+        # self.settings["collaborative_file_poll_interval"]
+        # self.settings["collaborative_document_save_delay"]
```

Comment:
Dead code

```suggestion
```

Reply:
No, I want to keep it there as comments. It helps to rapidly see what settings are available.